jungle/rds.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+import subprocess
+import sys
+
+import boto3
+import botocore
+import click
+
+
+def format_output(instances, flag):
+    """return formatted string per instance"""
+    out = []
+    line_format = '{0}\t{1}\t{2}\t{3}'
+    name_len = _get_max_name_len(instances) + 3
+    if flag:
+        line_format = '{0:<' + str(name_len+5) + '}{1:<16}{2:<65}{3:<16}'
+
+    for i in instances:
+        tag_name = i['DBInstanceIdentifier']
+        endpoint = "%s:%s" % (i['Endpoint']['Address'], i['Endpoint']['Port'])
```

Comment:
Could you use `format` for string format instead of `%`? I just want to make it consistent with other code.